Dockerfile.template

```diff
@@ -4,6 +4,7 @@ LABEL image="{{ pihole.name }}:{{ pihole.os }}_{{ pihole.arch }}"
 LABEL maintainer="{{ pihole.maintainer }}"
 LABEL url="https://www.github.com/diginc/docker-pi-hole"
 
+ARG DEBIAN_FRONTED=noninteractive
```

Comment:
this is probably not needed now (this was before I fixed the renaming of the apt command), and it should be an env not an ARG